src/screens/Home/OnboardingScreen/OnboardingScreen.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import HomeScreen from '../HomeScreen/HomeScreen'
+import {
+  Text,
+  View,
+  Image,
+  Button,
+  TouchableOpacity,
+  Alert
+} from 'react-native'
+import styles from './styles'
+
+/* AKA: Q&A screen */
+export default function OnboardingScreen(props) {
+  const {navigate} = props.navigation;
```

Comment:
same thing as mentioned before - the props should not be defined beforehand. this way React knows what to expect from props and so you can catch errors more easily. 

```
interface Props {
  navigation: NavigationScreenProp<NavigationState>;
}

export default function OnboardingScreen(props: Props) {}
```
And then you'll need to import NavigationScreenProp and NavigationState from react-navigation

Read about it here: https://reactnavigation.org/docs/en/next/typescript.html